core/chains/evm/txmgr/evm_tx_store.go

```diff
@@ -957,6 +957,16 @@ func (o *evmTxStore) FindReceiptsPendingConfirmation(ctx context.Context, blockN
 	return
 }
 
+func (o *evmTxStore) FindHighestSequence(ctx context.Context, fromAddress common.Address, chainId *big.Int) (nonce evmtypes.Nonce, err error) {
```

Comment:
Is this an EVM only signature? If not, would 'latest' or 'newest' be more general than 'highest'?

Reply:
Good point. This generalized for all chains so I can update it to `FindLatestSequence`